src/wp-includes/fonts.php

```diff
@@ -118,6 +118,16 @@ function wp_unregister_font_collection( string $slug ) {
  * }
  */
 function wp_get_font_dir( $defaults = array() ) {
+	/*
+	* Bail if the 'font_dir' filter is already running.
+	*
+	* This avoids an infinite loop that can occur
+	* if wp_upload_dir() is called inside a 'font_dir' callback.
+	*/
+	if ( doing_filter( 'font_dir' ) ) {
```

Comment:
```suggestion
	/*
	 * Bail if the 'font_dir' filter is already running.
	 *
	 * This avoids an infinite loop that can occur
	 * if wp_upload_dir() is called inside a 'font_dir' callback.
	 */
	if ( doing_filter( 'font_dir' ) ) {
```
When will IDEs correctly outdent multiline comments?! :joy: This drives me nuts when it happens to me!